utils/hono-influxdb-connector/README.md

```diff
@@ -25,25 +25,58 @@ To build the Hono-InfluxDB-Connector navigate to its directory.
 `cd utils/hono-influxdb-connector`
 
 This project is build using Gradle and has been tested with [Gradle v5.0](https://github.com/gradle/gradle/releases/tag/v5.0.0).
-To compile the project simply run.
 
-`gradle build`
+To build with Gradle, create a Docker image and push the image to a Docker registry, 
+run
+
+```bash
+./build.sh \
+    <Hostname of the Docker registry> \
+    <Username to log into the Docker registry> \
+    <Password to log into the Docker registry>
+```
+
+## Deployment
+
+To deploy in a Kubernetes cluster, run
+
+```bash
+./deploy.sh \
+    <Cluster-internal URL of InfluxDB, typically http://influxdb.hono:8086> \
+    <Cluster-internal hostname of QPID dispatch router, typically hono-dispatch-router-ext.hono> \
+    <Cluster-internal port of QPID dispatch router, typically 5671> \
```

Comment:
Why do we have first a URL (for the InfluxDB) and then the combination of hostname and port (QPID dispatch router). Could we only give a URL for the dispatch router as well?